params/config.go

```diff
@@ -108,6 +109,7 @@ var (
 		PetersburgBlock:     big.NewInt(4_939_394),
 		IstanbulBlock:       big.NewInt(6_485_846),
 		MuirGlacierBlock:    big.NewInt(7_117_117),
+		RamanujanBlock:      big.NewInt(0),
```

Comment:
This should probably left as nil and not zero (which would mean the this fork is activated straight away)